src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md

```diff
@@ -1,17 +1,53 @@
-# OpenTelemetry.Instrumentation.GrpcNetClient
+# Grpc.Net.Client Instrumentation for OpenTelemetry
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 
-Automatically instruments the outgoing requests from
-[Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client).
+This is an [Instrumentation Library](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/glossary.md#instrumentation-library)
+which instruments [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client)
+and collects telemetry about outgoing gRPC requests.
 
-## Installation
+## Steps to enable OpenTelemetry.Instrumentation.GrpcNetClient
+
+### Step 1: Install Package
+
+Add a reference to the
+[`OpenTelemetry.Instrumentation.GrpcNetClient`](https://www.nuget.org/packages/opentelemetry.instrumentation.grpcnetclient)
+package. Also, add any other instrumentations & exporters you will need.
 
 ```shell
 dotnet add package OpenTelemetry.Instrumentation.GrpcNetClient
 ```
 
+### Step 2: Enable Grpc.Net.Client Instrumentation at application startup
+
+Grpc.Net.Client instrumentation must be enabled at application startup. For
+example, if you are using an ASP.NET Core application, then this is typically
+done in the `ConfigureServices` of your `Startup` class.
+
+The following example demonstrates adding Grpc.Net.Client instrumentation to a
+console application. This example also sets up the OpenTelemetry Console
+exporter, which requires adding the package
+[`OpenTelemetry.Exporter.Console`](../OpenTelemetry.Exporter.Console/README.md)
+to the application.
+
+```csharp
+using OpenTelemetry.Trace;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        using Sdk.CreateTracerProviderBuilder()
+            .AddGrpcClientInstrumentation()
+            .AddConsoleExporter()
+            .Build();
+
+        // Perform gRPC invocations
```

Comment:
Ok, so this is admittedly cheating, but before getting too involved here I wanted to get some thoughts from others...

In order to have a fully functional example described here in the readme that would allow people to just copy and paste from here and have running code, we'd have to include a fair amount more details.

For example, the following assumes a proto file describing a `Greeter` service as well as a gRPC service hosting that service at `localhost:5001`.

```csharp
var channel = GrpcChannel.ForAddress("https://localhost:5001");
var client = new Greeter.GreeterClient(channel);
var reply = await client.SayHelloAsync(new HelloRequest { Name = "GreeterClient" });
```

I've been sitting on a sample gRPC client and server application I need to PR. One thought is to keep the example here in readme abbreviated and then link to the sample apps.

Reply:
IMO we can get the `examples` as an example (LOL). There, we are assuming that we have the zipkin running to export data. So, we just added a comment to run the docker and that's done. I think, here is the same thing, we can suppose that the user who tries this already know that he/she needs a working Grpc channel open.

Reply:
Yea, I'm of that mindset as well, though we do have some examples in readme